client/app/views/auth/index.coffee

```diff
@@ -35,6 +35,7 @@ module.exports = class AuthView extends LayoutView
 
     ui:
         passwd: 'input[type=password]'
+        authCode: 'input[type=text]'
```

Comment:
Can you rely on name attribute rather than text one (i.e. `'input[name=otp]'`? It's more understandable and will be more future-proof.